content/en/docs/examples/alb-nlb/index.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Securely direct traffic from IBM Cloud Kubernetes Service Ingress to Istio Ingress Gateway
+description: Configuring the IKS Ingress ALB to direct traffic Istio Ingress gateway with mutual TLS.
+weight: 43
+keywords: [traffic-management,ingress,file-mount-credentials,iks]
+---
+
+This example shows how to configure the [IBM Cloud Kubernetes Service](https://www.ibm.com/cloud/kubernetes-service/)
+[Ingress ALB](https://cloud.ibm.com/docs/containers?topic=containers-ingress-about) to direct traffic to the Istio
+ingress gateway with [NLB](https://cloud.ibm.com/docs/containers?topic=containers-loadbalancer-about), while securing
+the traffic between them using {{< gloss  >}}mutual TLS authentication{{< /gloss >}}.
+
+When you use IBM Cloud Kubernetes Service without Istio, you may control your ingress traffic using the provided
+Application Load Balancer (ALB). This ingress-traffic routing is configured using a Kubernetes
+[Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resource with
+[ALB-specific annotations](https://cloud.ibm.com/docs/containers?topic=containers-ingress_annotation).
+
+When you start using Istio in your IKS cluster, it is recommended that you use the
```

Comment:
did we explain what IKS stands for earlier?

Reply:
I will add a link to IKS.